Objects/rangeobject.c

```diff
@@ -122,14 +115,14 @@ range_new(PyTypeObject *type, PyObject *args, PyObject *kw)
             return NULL;
         default:
             PyErr_Format(PyExc_TypeError,
-                         "range expected at most 3 arguments, got %zd",
+                         "range expected at most 3 arguments, got %zu",
                          num_args);
```

Comment:
It should be reset to %zd, since num_args is signed, no?

Reply:
You're right.